test/flatten.js

```diff
@@ -50,6 +50,31 @@ describe('flatten', function () {
         });
       });
     });
+    describe('null', function () {
+      before(function () {
+        this.obj = {
+          num: new Number(1), // ignore
```

Comment:
The comment is out of date



—@tjmehta
Typed using my thumbs..

On Thu, Jul 16, 2015 at 9:26 AM, Bryan Kendall <notifications@github.com>
wrote:

>> @@ -50,6 +50,31 @@ describe('flatten', function () {
>>          });
>>        });
>>      });
>> +    describe('null', function () {
>> +      before(function () {
>> +        this.obj = {
>> +          num: new Number(1), // ignore
> well, I guess that `ignore` is an... interesting comment. it doesn't ignore it, it just uses it straight up (like the value at `foo.qux`). If I was looking at the tests for an example and saw this comment, I may come away confused (as I have)
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/tjmehta/keypather/pull/17/files#r34807591